docs/source/install.rst

```diff
@@ -73,17 +73,13 @@ Install Chainer from source
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The tarball of the source tree is available via ``pip download chainer`` or from `the release notes page <https://github.com/chainer/chainer/releases>`_.
-You can use ``setup.py`` to install Chainer from the tarball::
+You can install Chainer from the tarball::
 
-  $ tar zxf chainer-x.x.x.tar.gz
-  $ cd chainer-x.x.x
-  $ python setup.py install
+  $ pip install chainer-x.x.x.tar.gz
 
 You can also install the development version of Chainer from a cloned Git repository::
 
-  $ git clone https://github.com/chainer/chainer.git
-  $ cd chainer
-  $ python setup.py install
+  $ pip install git+https://github.com/chainer/chainer.git
```

Comment:
For git installation, users may want to modify Chainer code.
How about keeping cloning part? i.e.
```
$ git clone https://github.com/chainer/chainer.git
$ cd chainer
$ pip install .
```